### ISSUE TITLE
Possible clade misannotations

### ISSUE THREAD

**Issue by theosanderson** [nextstrain/monkeypox#6]
Thank you very much for this resource.

On my very naive tree, the following sequences cluster with West African sequences:

```
MPXV_WRAIR7_61__Walter_Reed_267
COP_58
Liberia_1970_184
Ivory_Coast_2012
Sierra_Leone
USA_2003_044
USA_2003_039

added 23/5:
MPXV_TNP_2017_North_Ponan
3030
```

though they are annotated as `CA`. In every case I have so far looked into in the literature, a West Africa-clade annotation would seem reasonable to my non-expert eyes. (But I may well be wrong in terms of how you choose to define the clades, etc.)

Thanks again

**Comment by theosanderson** [comment]
Unrelatedly, but to avoid spamming issues, one suggestion would be to consider excluding apostrophes from strain names as I think happens for SARS-CoV-2

**Comment by theosanderson** [comment]
With a slightly less naive tree than I had before I get 
![image](https://user-images.githubusercontent.com/19732295/169874537-cd984840-c858-49ad-98ca-8fe1118d66c4.png)
with orange indicating unexpected `CA` annotations

**Comment by rneher** [comment]
thanks Theo. I just merged PR #7 by @babarlelephant that fixed most of these issues. And fixed one more that you pointed out. We'll hopefully have a proper clade naming system soon. We'll also add an build containing the full diversity.
